### ISSUE TITLE
Migrate article: AWS Privatelink setup step-by-step, using Aiven and AWS web console

### ISSUE THREAD

**Issue by lornajane** [aiven/devportal#823]
Migrate the content from this article to fit the documentation portal format https://help.aiven.io/en/articles/5858370

(Assign this issue to yourself before you start, and remember to mention the issue number when you open your pull request)

**Comment by lornajane** [comment]
@fmanimali are you still working on this one?

**Comment by fmanimali** [comment]
I think  the article has been migrated and the pr closed, if this is the case, the issue can be closed

**Comment by lornajane** [comment]
I don't believe this is closed, I think the PR is #1440 and this is waiting for some feedback from either @fmanimali or @ftisiot.
